### DISCUSSION TITLE
NVDEC / NVENC support?

### DISCUSSION THREAD

**Discussion by rcastill** [rust-av/rust-av#178] [General]
Is there any support for hardware acceleration? specifically for NVIDIA chipsets?

**Comment by lu-zero** [comment]
Not yet, I started writing the bindings for it long ago but I had other tasks filling my spare time =/

**Reply by rcastill** [reply]
I understand. Where's what you started?

**Reply by Luni-4** [reply]
@rcastill https://github.com/rust-av/nvidia-video-codec-rs
